Sources/Fullscreen/NotificationCenter/PushNotificationDetailsView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public enum PushNotificationDetails {
```

Comment:
Same here as in network code, probably should not include "push" in the name

Reply:
Yep, I'll change this